client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -495,34 +496,38 @@ export class PlanFeatures2023Grid extends Component<
 	}
 
 	renderPlanPrice( planPropertiesObj: PlanProperties[], options?: PlanRowOptions ) {
-		const { isReskinned, isLargeCurrency } = this.props;
+		const { isReskinned, isLargeCurrency, translate } = this.props;
 
-		return planPropertiesObj
-			.filter( ( { isVisible } ) => isVisible )
```

Comment:
Sorry, I did not notice this earlier! I was wondering if the removal of `.filter( ( { isVisible } ) => isVisible )` was intentional. Same for L750 below.